pipelines/skylab/slideseq/test_inputs/Scientific/Puck_200501_23.mm10.fastqs.json

```diff
@@ -0,0 +1,11 @@
+{
```

Comment:
We do not want to keep most of the scientific test jsons. Only one should be there. See ticket PD-2092 on jira.